libs/devops/src/lib/firestoreMigrations/0084.ts

```diff
@@ -22,11 +28,43 @@ export async function upgrade(db: Firestore) {
     // Update Certifications
     const movie = doc.data() as Movie;
 
-    if (!movie?.certifications.length) return false;
-
     movie.certifications = movie.certifications.filter(c => !['artEssai', 'awardedFilm', 'aListCast'].includes(c));
 
+    // Remove atributes: 'reviews', 'ratings', 'boxoffice'
+    if ((movie as any)?.boxoffice) delete (movie as any).boxoffice;
+    if ((movie as any)?.reviews) delete (movie as any).reviews;
+    if ((movie as any)?.ratings) delete (movie as any).ratings;
```

Comment:
In fact, this should be enough : 
```suggestion
   delete (movie as any).boxoffice;
   delete (movie as any).reviews;
   delete (movie as any).ratings;
```